src/core/types.ts

```diff
@@ -23,3 +25,80 @@ export declare type GetSupportedVoicesResponse = {
   styleList: string[];
   wordsPerMinute: string;
 }[];
+
+export type TranscriberMessage = {
```

Comment:
suggestion: name them something like `DataMessage` and `ChatMessage` to differentiate from livekit messages and user/assistant messages

Reply:
https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/10/commits/931f36f53527b5792942f48d93a4e215a25203d9